bootstrap/scripts/2-download.sh~

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
This is also a temporary file that shouldn't be here.

Reply:
yeup